src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -188,12 +204,13 @@ def __call__(self,
         Returns:
           A Call object instance which is an awaitable object.
         """
+        if metadata is None:
+            metadata = _IMMUTABLE_EMPTY_TUPLE
```

Comment:
(nit) alternatively, we can make ``_IMMUTABLE_EMPTY_TUPLE`` the default value for the ``metadata`` argument, WDYT?

Reply:
Updated as suggested.